packages/core/src/sass/app/_common/components/cookie-banner.scss

```diff
@@ -0,0 +1,96 @@
+/** @define animation */
+@keyframes FadeInDown {
+    from {
+        opacity: 0;
+        bottom: -208px;
+        @include mobile() {
+            bottom: -168px;
+        }
+    }
+    to {
+        opacity: 0.72;
+        bottom: 0;
+    }
+}
+@keyframes FadeOutUp {
+    from {
+        opacity: 0.72;
+        bottom: 0;
+    }
+    to {
+        opacity: 0;
+        bottom: -208px;
+        @include mobile() {
+            bottom: -168px;
+        }
+    }
+}
+/** @define cookie-banner; weak */
+.cookie-banner {
+    position: fixed;
+    z-index: 10;
+    text-align: center;
+    background-color: var(--general-section-1);
+    color: var(--text-general);
+    font-size: 16px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: 1.5;
+    letter-spacing: normal;
+    padding: 4.8rem 0;
+    bottom: 0;
+    width: 100%;
+    height: 208px;
+    transition: all 0.3s ease-in-out;
```

Comment:
Let's only apply the transitions that are used instead of `all`?

Reply:
Used transform instead of animation.